lib/gems/pending/appliance_console.rb

```diff
@@ -597,7 +600,9 @@ module ApplianceConsole
 
       when I18n.t("advanced_settings.scap")
         say("#{selection}\n\n")
-        ApplianceConsole::Scap.new.lockdown
+        ApplianceConsole::Scap.new(
+          :rules_dir => File.expand_path("productization/appliance_console/config", RAILS_ROOT)
+        ).lockdown
```

Comment:
@Fryguy I agree. Not sure how much it will help but, I can put this path in a constant (like `SCAP_RULES_DIR`) at the top of the file and just reference that here. What do you think?